elements/b.html

```diff
@@ -0,0 +1,44 @@
+<article class="article">
+  <header class="article-header">
+    <h2>
+      <code>
+        &lt;b&gt;
+      </code>
+    </h2>
+    <div class="base-words">
+      <div class="base-word-line">
+        English Base: <strong>Boldface Text</strong>
+      </div>
+      <div class="base-word-line">
+        Spanish translation/Traducción en Español: <strong></strong>
+      </div>
+    </div>
+  </header>
+  <div class="definition">
+    <div class="definition-box definition-spanish">
+      <h4>Definici&oacute;n</h4>
+      <p></p>
+    </div>
+    <div class="definition-box definition-english">
+      <h4>Definition</h4>
+      <p>Originally, &lt;b&gt; was meant to make text boldface.  Currently, it is used to indicate a section of text that is styled differently from other text.  The &lt;b&gt; element is usually used when other command codes don't work well for the situation.  It is also used to indicate keywords, product names, and other text that need to stand out but do not have any emphasis on the text.  It is strongly recommended to use &lt;em&gt;, &lt;strong&gt; or &lt;mark&gt; when indicating important text.</p>
```

Comment:
Does that work better?  I want to mention the others so people know what options there are.